src/main/java/com/minecolonies/coremod/colony/requestsystem/management/IStandardRequestManager.java

```diff
@@ -21,22 +21,22 @@
 public interface IStandardRequestManager extends IRequestManager
 {
     @NotNull
-    BiMap<IToken, IRequestResolverProvider> getProviderBiMap();
+    BiMap<IToken<?>, IRequestResolverProvider> getProviderBiMap();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1452)